addons/web/models/ir_ui_menu.py

```diff
@@ -1,41 +1,52 @@
-from odoo import api, models, tools
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
 
+from odoo import models, tools
 
 class IrUiMenu(models.Model):
-
     _inherit = "ir.ui.menu"
 
-    @api.model
     @tools.ormcache_context('self._uid', 'debug', keys=('lang',))
```

Comment:
load_menus is already orm_cached. So maybe the caching here is useless (we only prevent the for loop BUT we store a specific cache for it....)